pkg/api/common.go

```diff
@@ -101,6 +105,12 @@ func ParseFlags(fs *flag.FlagSet, cfg *Config) *Config {
 	fs.StringVar(&cfg.Auth.BasicAuthPasswordFile, "auth-password-file", "", "Path for auth password file containing the actual password used for web endpoint authentication. This flag should be set together with auth-username. It is mutually exclusive with auth-password and bearer-token methods.")
 	fs.StringVar(&cfg.Auth.BearerToken, "bearer-token", "", "Bearer token (JWT) used for web endpoint authentication. Disabled by default. Mutually exclusive with bearer-token-file and basic auth methods.")
 	fs.StringVar(&cfg.Auth.BearerTokenFile, "bearer-token-file", "", "Path of the file containing the bearer token (JWT) used for web endpoint authentication. Disabled by default. Mutually exclusive with bearer-token and basic auth methods.")
+
+	// PromQL configuration flags.
+	fs.DurationVar(&cfg.MaxQueryTimeout, "promql-query-timeout", 2*time.Minute, "Maximum time a query may take before being aborted. Note: The 'timeout' parameter in "+
+		"'/api/v1/query.*' should be within 'promql-query-timeout' range in order to be respected.")
+	fs.DurationVar(&cfg.SubQueryStepInterval, "promql-subquery-step-interval", 1*time.Minute, "Step interval to be used for PromQL subquery evaluation evaluation, "+
```

Comment:
should this be promql-default-subquery-step-interval?

Reply:
Yeah, that would be better